index.html

```diff
@@ -290,7 +294,19 @@ <h3>MathML Element Mappings</h3>
 		  </td>
 		  <td class="axapi">
 		    <span class="role">AXRole: <code>NSAccessibilityGroupRole</code></span><br />
-		    <span class="subrole">AXSubrole: <code>AXMathMultiscript</code></span>
+		    <span class="subrole">AXSubrole: <code>AXMathMultiscript</code></span><br />
```

Comment:
The mmultiscripts element is a bit a mess. It's a bit difficult for me to review but see https://mathml-refresh.github.io/mathml-core/#prescripts-and-tensor-indices-mmultiscripts for the valid structure.

Reply:
I hope the structure reflects spec here, but not sure on object-c format used to represent data. I tried to follow webkit implementation here (this bug has some webkit code links https://bugs.chromium.org/p/chromium/issues/detail?id=1052425&q=label%3Aa11y-mathml-mac&can=2), but since I don't feel very confident in object-c, it'd be good to have somebody (you?) to make a sanity check.

Reply:
So the general structure is

    <mmultiscripts>
    BASE
    postsubscript1
    postsupscript1
    postsubscript2
    postsupscript2
    ...
    postsubscriptN
    postsupscriptN
    <mprescripts/>
    presubscript1
    presupscript1
    presubscript2
    presupscript2
    ...
    presubscriptM
    presupscriptM
    </mmultiscripts>

Note that `<mprescripts/>` is empty it's just a delimiter between post and prescripts (yeah, I told you this mathml 3 syntax is not great).

Checking what WebKit does, the AXMathPrescripts and AXMathPostscripts attribute returns an NSArray of NSMutableDictionarys. Each NSMutableDictionary has two keys AXMathSubscript and AXMathSuperscript. So using JSON notation, it's:

    AXMathPrescripts =
    [{AXMathSubscript: presubscript1, AXMathSuperscript: presupscript1},
     {AXMathSubscript: presubscript2, AXMathSuperscript: presupscript2},
     {AXMathSubscript: presubscript3, AXMathSuperscript: presupscript3},
     ...
     {AXMathSubscript: presubscriptN, AXMathSuperscript: presupscriptN}]
    
    AXMathPostcripts =
    [{AXMathSubscript: postsubscript1, AXMathSuperscript: postsupscript1},
     {AXMathSubscript: postsubscript2, AXMathSuperscript: postsupscript2},
     {AXMathSubscript: postsubscript3, AXMathSuperscript: postsupscript3},
     ...
     {AXMathSubscript: postsubscriptN, AXMathSuperscript: postsupscriptN}]